templates/test.template

```diff
@@ -2,21 +2,20 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-template: default
 author: Mathias Bynens
-desc: Unicode property escapes `\p{<%= UnicodePropertyValueExpression %>}` and `\P{<%= UnicodePropertyValueExpression %>}`
+description: >
+  Unicode property escapes `\p{}` and \P{}` for `<%= UnicodePropertyValueExpression %>`
 esid: sec-static-semantics-unicodematchproperty-p
-flags: [generated]
```

Comment:
Shouldn’t we signify somehow that these tests shouldn’t be edited manually? I’ve added a link to this repository in the Info section — maybe that’s enough?